ios/CustomerioReactnative.swift

```diff
@@ -12,8 +13,12 @@ class CustomerioReactnative: NSObject {
     /**
      Initialize the package before sending any calls to the package
      */
-    @objc(initialize:apiKey:region:configData:pversion:)
-    func initialize(siteId: String, apiKey: String, region :String, configData: Dictionary<String, AnyHashable>, pversion: String) -> Void {
+    @objc(initialize:configData:pversion:)
+    func initialize(env: Dictionary<String, AnyHashable>, configData: Dictionary<String, AnyHashable>, pversion: String) -> Void {
+        
+        guard let siteId = env["siteId"] as? String, let apiKey = env["apiKey"] as? String, let region = env["region"] as? String, let organizationId = env["organizationId"] as? String else {
+            return
+        }
```

Comment:
Not very good in swift but just wanted to check will this still work if `organizationId` isn't present? 😅  As we have made it optional now 👀

Reply:
The default value of `organizationId` is "" which is set in this [line](https://github.com/customerio/customerio-reactnative/blob/c87ef4200b936152f3a58410bafc7beea00fb964/src/CustomerioConfig.tsx#L26). The code will not break in any case.

In case, user doesn't set organizationId then in-app will not get initialized. Check [here](https://github.com/customerio/customerio-reactnative/blob/c87ef4200b936152f3a58410bafc7beea00fb964/ios/CustomerioReactnative.swift#L32)

Reply:
Perfect, thanks for the explanation 👍🏻